components/proposal-merge.tsx

```diff
@@ -0,0 +1,152 @@
+import { useEffect, useState } from "react";
+
+import { useTranslation } from "next-i18next";
+
+import LockedIcon from "assets/icons/locked-icon";
+
+import calculateDistributedAmounts from "helpers/calculateDistributedAmounts";
+import { formatNumberToCurrency } from "helpers/formatNumber";
+import { handlePercentage } from "helpers/handlePercentage";
+
+import { ProposalExtended } from "interfaces/bounty";
+
+import useApi from "x-hooks/use-api";
+
+import BountyDistributionItem from "./bounty-distribution-item";
+import Button from "./button";
+import Modal from "./modal";
+
+interface distributedAmounts {
+  treasuryAmount: number;
+  mergerAmount: number;
+  proposerAmount: number;
+  proposals: number[];
+}
+
+interface props {
+  amountTotal: number,
+  tokenSymbol?: string,
+  proposal: ProposalExtended,
+  mergeBounty: () => void;
+  canMerge: boolean
+}
+
+export default function ProposalMerge({
+  amountTotal,
+  tokenSymbol,
+  proposal,
+  mergeBounty,
```

Comment:
switch naming to `onClickMerge` so we can understand that this is both a function _and_ a "event handler"